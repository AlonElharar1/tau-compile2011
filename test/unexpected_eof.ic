/* this is a comment that doens't end until the file is over.
this should result in a lexical error.