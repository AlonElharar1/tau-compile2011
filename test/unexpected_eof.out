Line 2: Unexpected EOF
